mlflow/server/js/src/experiment-tracking/components/MetricsPlotControls.js

```diff
@@ -245,7 +245,7 @@ const styles = {
   xAxisControls: (theme) => ({
     label: { marginTop: theme.spacing.xs, marginBottom: theme.spacing.xs },
   }),
-  controlsWrapper: { minWidth: 300 },
+  controlsWrapper: { minWidth: 300, maxWidth: 500 },
```

Comment:
Ideally it would be awesome if we could do % or em instead of px. That would help with different sizes of screens. What do you think?

Reply:
Sounds good! I'll try %.